### ISSUE TITLE
Add option to print duration in present tense

### ISSUE THREAD

**Issue by mklein994** [imp/chrono-humanize-rs#1]
Currently, If I specify a duration, it acts as a moment in time. Could there be some way to format the duration in present tense, regardless of how long it is?

Currently:
```rust
use chrono::Duration;
use chrono_humanize::HumanTime;

println!(
    "{:#}\n{:#}",
    HumanTime::from(Duration::seconds(242)),
    HumanTime::from(Duration::seconds(-242))
);

//=>  in 4 minutes and 2 seconds
//=>  4 minutes and 2 seconds ago
```
What I'm looking for:
```rust
// (something that says to format in present tense...)

//=>  4 minutes and 2 seconds
```
I hope this describes enough.

P.S. Thanks for this crate. 👍

**Comment by imp** [comment]
@mklein994 Glad you find this crate useful. Let me see how to best expose this functionality.
Note, that this repo is a mirror of the upstream hosted at https://gitlab.com/imp/chrono-humanize-rs, where this is issue https://gitlab.com/imp/chrono-humanize-rs/issues/6

**Comment by imp** [comment]
@mklein994 Fixed in newly released 0.0.11 that provides `HumanTime::to_text_en()`.
See usage example here - https://docs.rs/chrono-humanize/0.0.11/chrono_humanize/
